plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/BaseDeltaLakeMinioConnectorTest.java

```diff
@@ -971,4 +1357,32 @@ private List<String> getTableFiles(String tableName)
                 .map(path -> format("s3://%s/%s", bucketName, path))
                 .collect(toImmutableList());
     }
+
+    private List<MaterializedRow> getCdfColumnsWithoutTimestamp(MaterializedResult materializedResult)
```

Comment:
use `io.trino.sql.query.QueryAssertions.QueryAssert#exceptColumns`
and generally switch to query-based assertions (`assertThat(query(...))`)